src/common/rbac/project_evaluator.go

```diff
@@ -25,7 +27,7 @@ import (
 )
 
 // NewProjectRBACEvaluator returns permission evaluator for project
-func NewProjectRBACEvaluator(ctx security.Context, pm promgr.ProjectManager) evaluator.Evaluator {
+func NewProjectRBACEvaluator(user *models.User, pm promgr.ProjectManager) evaluator.Evaluator {
```

Comment:
I think it'd better be called `NewProjectUserEvaluator` or `NewProjectRBACUserEvalauator` 
@heww  thoughts?